pkgs/development/compilers/openjdk/generic.nix

```diff
@@ -59,9 +60,9 @@
   openjfx23,
   openjfx_jdk ?
     {
-      "17" = openjfx17;
-      "21" = openjfx21;
-      "23" = openjfx23;
+      "17" = openjfx17.__spliced.buildBuild or openjfx17;
+      "21" = openjfx21.__spliced.buildBuild or openjfx21;
+      "23" = openjfx23.__spliced.buildBuild or openjfx23;
```

Comment:
I’m not sure this is right – OpenJFX is just here to get its path embedded into the resulting JDK and the native binary parts of it are run on the host.

(Actually, embedding OpenJFX is sorta‐kinda‐deprecated and I’d like to get rid of it for 25.05 in favour of treating it like a normal dependency, but that’s another matter.)

Reply:
Remove the left side of the or